packages/gatsby/cache-dir/production-app.js

```diff
@@ -85,13 +89,59 @@ apiRunnerAsync(`onClientEntry`).then(() => {
                   />
                 </Router>
               </ScrollContext>
+              <RouteAnnouncer />
             </RouteUpdates>
           )}
         </EnsureResources>
       )
     }
   }
 
+  const RouteAnnouncer = () => {
+    const [announcement, setAnnouncement] = useState(``)
+    useEffect(() => {
+      announceLocation = () => {
+        requestAnimationFrame(() => {
+          let pageName = `new page`
+          if (document.title) {
+            pageName = document.title
+          }
+          const pageHeadings = document
+            .getElementById(`gatsby-focus-wrapper`)
+            .getElementsByTagName(`h1`)
+          if (pageHeadings) {
+            pageName = pageHeadings[0].innerHTML
```

Comment:
Might be better to use textContent here to avoid potentially reading HTML tags

```suggestion
            pageName = pageHeadings[0].textContent
```

Reply:
I do wonder if it wouldn't be safer to use `innerText` - see https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Differences_from_innerText for differences

downside is that accessing it cause [reflow](https://developer.mozilla.org/en-US/docs/Glossary/Reflow) ... :(